pkg/build/admission/defaults/admission.go

```diff
@@ -83,6 +83,15 @@ func (a *buildDefaults) applyBuildDefaults(build *buildapi.Build) {
 		addDefaultEnvVar(envVar, buildEnv)
 	}
 
+	sourceDefaults := a.defaultsConfig.SourceStrategyDefaults
+	sourceStrategy := build.Spec.Strategy.SourceStrategy
+	if sourceDefaults != nil && sourceDefaults.Incremental != nil && *sourceDefaults.Incremental &&
```

Comment:
Seems weird to have a default that I can only set one way.  Builds are mostly read-only, right?  Seems like I should be allowed to default it to `false` if I want.